fedimint-build/src/envs.rs

```diff
@@ -3,6 +3,4 @@ pub const FORCE_GIT_HASH_ENV: &str = "FEDIMINT_BUILD_FORCE_GIT_HASH";
 
 /// Env variable the cargo will set during crate build to pass the detected git
 /// hash to the binary itself.
-// Note: Keep in sync with `fedimint_core::envs`, which is not reused-to avoid
```

Comment:
Why remove a comment?

Reply:
Because we removed the matching const in `fedimint-core/src/envs.rs`